### PR TITLE
toml: convert yaml value checks in iarna test suite

### PR DESCRIPTION
**Pull request by Larpon**
This PR will enable support for converting the YAML value check files to JSON so we can validate a little more values from the iarna test suite.

### CODE REVIEW

**Comment by Larpon** [comment]
Right I think we've hit a memory limit or something - will fix ASAP

File: .github/workflows/toml_ci.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   toml-module-pass-external-test-suites:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-18.04
```
Comment (spytheman) [inline .github/workflows/toml_ci.yml:13]: why?

**Comment by spytheman** [comment]
hm, why bother with yaml test suites at all?

File: .github/workflows/toml_ci.yml (hunk 1 above)
Comment (Larpon) [reply .github/workflows/toml_ci.yml:13]: `jq` [is failing](https://github.com/vlang/v/runs/4367807556?check_suite_focus=true#step:11:152) on 20.04 for some unknown reason

**Comment by Larpon** [comment]
> hm, why bother with yaml test suites at all?

Checking that values are as they should be is a good thing IMO even if it requires some more tweaking.
On the other hand it has only revealed 2 bugs in the `toml` module. I'm fine with letting it skip.

I'll see if I can fix the CI for now - and I'll let you decide if it goes in :slightly_smiling_face:

**Comment by Larpon** [comment]
I've fixed the issues with python.

... I get that the conversion process gives false positives and adds complexity to the specific test suite.
I'm just not sure about skipping them is a good idea - it's still ~114 tests we didn't have before :thinking: 
We'd also miss any future updates - although the BurntSushi tests seem more active at the moment.

**Pull request merged**